static/src/components/PlayerDisplay.js

```diff
@@ -26,9 +26,12 @@ export class PlayerDisplay extends Component {
     constructor(props) {
         super(props);
         this.state = {
+            userID: null,
             displayID: null,
             displayData: null,
-            userID: null,
+            leftData: null,
+            rightData: null,
+            totalPlayers: null,
```

Comment:
do we ever use these state variables?

Reply:
Not in this PR.
The idea was to 'hold' the neighbours data so that if the current player could buy some material from the left neighbour, there will be a left arrow next to their quantity of that material to indicate an option to buy... (this is a "nice to have option").
- For now, I will remove it.